test/integration/suites/datastore-mysql-replication/01-test-variants

```diff
@@ -136,14 +136,8 @@ test-mysql-replication() {
     readonly_root_password=$(get_mysql_root_password "${readonly_service_name}")
 
     # Use:
-    # - regular replication for MySql 5.5 and 5.6.
     # - group replication for MySql 5.7 and 8.0
     case $service_prefix in
```

Comment:
Should we remove this case statement and unconditionally configure for group replication since that is supported by all versions under test?

Reply:
Thanks! Thought I caught that one :-)

Reply:
I considered that; I don't think we need to check for unsupported versions since we explicitly invoke them below. We can always add a conditional back in if ever needed.